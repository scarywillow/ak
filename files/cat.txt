  section_program
  start:
    IN
    OUT R4
    HLT
