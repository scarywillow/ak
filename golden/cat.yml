in_source: |
  section_program
  start:
    IN
    OUT R4
    HLT
in_stdin: '"C","a","t"'
out_code: |
  Instruction counter: 0, IN []
  Instruction counter: 1, OUT ['R4']
  Instruction counter: 2, HLT []
out_stdout: 'Cat'
out_log: |
  DEBUG    root:machine.py:250 IN [] {PC: 0, REG0: 0, REG1: , REG2: , REG3: , REG4: , REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: []}
  DEBUG    root:machine.py:205 IN [] {PC: 0, REG0: 0, REG1: , REG2: , REG3: , REG4: C, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: []}
  DEBUG    root:machine.py:205 IN [] {PC: 0, REG0: 0, REG1: , REG2: , REG3: , REG4: Ca, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: []}
  DEBUG    root:machine.py:205 IN [] {PC: 0, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: []}
  DEBUG    root:machine.py:250 OUT ['R4'] {PC: 1, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: []}
  DEBUG    root:machine.py:198 OUT ['R4'] {PC: 1, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: ['C']}
  DEBUG    root:machine.py:198 OUT ['R4'] {PC: 1, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: ['C', 'a']}
  DEBUG    root:machine.py:198 OUT ['R4'] {PC: 1, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: ['C', 'a', 't']}
  DEBUG    root:machine.py:250 HLT [] {PC: 2, REG0: 0, REG1: , REG2: , REG3: , REG4: Cat, REG5: , REG6: , REG7: , REG8: , REG9: , REG10: , REG11: , REG12: , REG13: , REG14: , REG15: , OUTPUT_PORT: ['C', 'a', 't']}
